### PR TITLE
Add fact db_is_master

### PR DESCRIPTION
**Pull request by jnathlich**
Add a fact to determine if node is master.  Tiny add, helpful in configuring masters vs. replicas.

### CODE REVIEW

File: lib/facter/db_is_master.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Facter.add('db_is_master') do
+  setcode do
+    Facter::Core::Execution.exec('sudo -u postgres psql -t aristamd -c "select pg_is_in_recovery();" | /bin/grep -qi "f" && echo "true" || echo "false"')
```
Comment (antaflos) [inline lib/facter/db_is_master.rb]: What is `aristamd`?

**Comment by jnathlich** [comment]
Wow!
In that particular instance, aristamd is an example of me being 
extraordinarily stupid and has no place in that statement.

It's our company name, and (obviously, I suppose) a table name in the 
database we use internally in a cluster.

I'll remove the '-t aristamd' from the statement.

Sorry!

-Jim

------ Original Message ------

File: lib/facter/db_is_master.rb (hunk 1 above)
Comment (jnathlich) [reply lib/facter/db_is_master.rb]: In this case?  An artifact I forgot to take out... sorry about that!

**Comment by jnathlich** [comment]
-t switch removed from fact.

**Comment by jnathlich** [comment]
This seems to be failing in an area completely separate from my changes.
I did a squash and rebase to clean up and make sure I'm running from latest code.
Error is related to which repo is loaded vs what OS it detects... my change is a tiny add of one .rb file.
What are my next steps to track down and fix this?

**Comment by pmcmaw** [comment]
@jnathlich Travis now running green, apologies the failure was on our end and it has been resolved. 
Code looks great, if you could add a few unit tests I will get this merged for you 👍

**Comment by david22swan** [comment]
@jnathlich I apologize but due to the lack of movement on this PR I am now closing it. If at a later time you find that you can return to and complete it we will be more than happy to once again review it.

**Comment by jnathlich** [comment]
Yes, thank you!
My apologies...got completely buried after the original push.

------ Original Message ------

**Pull request closed without merging**